src/configs/cards.clj

```diff
@@ -0,0 +1,12 @@
+(ns configs.cards)
+
+(def card-map
```

Comment:
It's very nice to have all this in one place, and on top of it it's 100% freezable so we can have it in a database in the future (although I doubt that will ever be necessary or useful)

Reply:
It'll be both necessary and useful in the future: Having cards on a database will let us apply card-balancing without having to update the backend. Maybe even without updating the frontend either.

Still, I wouldn't do it right now, since at the moment it's easier to modified it here for testing than having to change a database.